README.md

```diff
@@ -39,31 +41,36 @@ conda install -c conda-forge -c rapidsai-nightly cuspatial
 
 To build and install cuSpatial from source:
 
+### Pre-requisite
+
+- gcc >= 7.5
+- cmake >= 3.23
+- miniconda
+
+### Fetch cuSpatial repository
+
+```shell
+export `CUSPATIAL_HOME=$(pwd)/cuspatial` && \
+git clone --recurse-submodules https://github.com/rapidsai/cuspatial.git $CUSPATIAL_HOME
```

Comment:
Do we need the submodule option? Do we still have submodules? We got rid of them in other repos.

Reply:
@isVoid can you answer this?

Reply:
Somehow this PR fall through my tracks. No I don't think we have submodules in the repo.